src/test/java/core/basesyntax/SalaryInfo.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+        LocalDate startDate = LocalDate.parse(dateFrom, DATE_FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo, DATE_FORMATTER);
```

Comment:
Ensure that the `dateFrom` and `dateTo` parameters are correctly formatted as 'dd.MM.yyyy'. If they are not, `LocalDate.parse` will throw a `DateTimeParseException`. Consider adding validation or exception handling to manage incorrect date formats.